helion/runtime/settings.py

```diff
@@ -87,6 +87,7 @@ class _Settings:
     )
     autotune_precompile: bool = sys.platform != "win32"
     print_output_code: bool = os.environ.get("HELION_PRINT_OUTPUT_CODE", "0") == "1"
+    print_triton_repro: bool = os.environ.get("HELION_PRINT_TRITON_REPRO", "0") == "1"
```

Comment:
Wonder should `HELION_PRINT_OUTPUT_CODE=1` just also print the call function? That way we might be able to save one environment variable too.